### DISCUSSION TITLE
Collect CVSS metadata retroactively or ping curator about it

### DISCUSSION THREAD

**Discussion by pinkforest** [rustsec/advisory-db#1350] [Ideas]
Is there a way to either retroactively fill CVSS metadata EDIT: - in automated / mechanical way

Or alternatively where we can provide a mechanism to ping the curator(s) to put this in automatically potentially ?

Discussion - continued from here: https://github.com/rustsec/advisory-db/issues/248

**Comment by tarcieri** [comment]
Yes, it can be added/updated at any time

**Reply by pinkforest** [reply]
I should clarify I meant more in automation sense that could give us actionable item in tooling wise
EDIT: edited at top
